src/decorations/functions.jl

```diff
@@ -23,7 +23,7 @@ const bool_binary_functions = (
 )
 
 for f in bool_functions
-    @eval $(f)(xx::DecoratedInterval) = $(f)(interval(xx))
+    @eval $(f)(xx::DecoratedInterval) = $(f)(xx.interval)
```

Comment:
Question: why do you prefer to use `xx.interval` instead of the helper function `interval(xx)`?